apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.cs

```diff
@@ -66,10 +71,8 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
         settings.Validate();
         // These values are validated in Settings.Validate() above, so no need to re-validate here.
         _modifyDeadlineSeconds = (int)((settings.AckDeadline ?? DefaultAckDeadline).TotalSeconds);
-        // FIXME: This should be named autoExtendDelay. 
         var autoExtendInterval = TimeSpan.FromSeconds(_modifyDeadlineSeconds) - (settings.AckExtensionWindow ?? DefaultAckExtensionWindow);
-        // FIXME: This should be named autoExtendDelayForExactlyOnceDelivery.
-        // FIXME: MinimumAckExtensionWindowForExactlyOnceDelivery should be renamed to MinimumLeaseExtensionDelay or MinimumLeaseExtensionSchedule with the default value of 60 seconds.
+        // FIXME: MinimumAckExtensionWindowForExactlyOnceDelivery is incorrectly named. It should be renamed to MinimumLeaseExtensionDelay or MinimumLeaseExtensionSchedule with the default value of 60 seconds.
```

Comment:
I'd still expect both of these comments to be just one of "FIXME: This is assigning a window to an interval, which is prohibited".
We don't need to go into the impact of this yet - it's just working out what doesn't make sense, so we can then revisit.